src/Caching/SqlServer/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,26 @@
+﻿Microsoft.Extensions.Caching.SqlServer.SqlServerCache
```

Comment:
These APIs shipped or mostly shipped in 5.0. Move everything that isn't new for 6.0 to the PublicAPI.Shipped.txt file.

Also, add the `#nullable enable` line then recreate this file to get the right annotations.